<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер расписания</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QGroupBox" name="scheduleGroupBox">
      <property name="title">
       <string>Распиание</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="findLabel">
         <property name="text">
          <string>Найти всех преподавателей группы:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="7">
        <widget class="QTableView" name="scheduleTableView"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="findPushButton">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="addNotePushButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="delNotePushButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="entityComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="data">
      <property name="title">
       <string>Данные</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="professorsGroupBox">
         <property name="title">
          <string>Преподаватели</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableView" name="professorsTableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="studyGroupsGroupBox">
         <property name="title">
          <string>Учебные группы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableView" name="studyGroupsTableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="subjectsGroupBox">
         <property name="title">
          <string>Предметы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableView" name="subjectsTableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="subjectTypesGroupBox">
         <property name="title">
          <string>Типы предметов</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QTableView" name="subjectTypesTableView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
